client/main.go

```diff
@@ -16,6 +16,7 @@ func main() {
 	conn, err := grpc.Dial("localhost"+port, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	if err != nil {
 		log.Fatalf("Did not connect: %v", err)
+		log.Fatal("Unable to connect to localhost", err)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a redundancy in logging the error when the connection fails. The `log.Fatalf` function already logs the error and then calls `os.Exit(1)`, so the program will not reach the `log.Fatal` line. You should remove the redundant log statement.
```diff
- 		log.Fatal("Unable to connect to localhost", err)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->